source/MapPlanetCard.cpp

```diff
@@ -44,10 +44,26 @@ MapPlanetCard::MapPlanetCard(const StellarObject &object, unsigned number, bool
 	hasShipyard = planet->HasShipyard();
 	hasOutfitter = planet->HasOutfitter();
 
-	reputationLabel = !hasSpaceport ? "No Spaceport" :
-		GameData::GetPolitics().HasDominated(planet) ? "Dominated" :
-		planet->GetGovernment()->IsEnemy() ? "Hostile" :
-		planet->CanLand() ? "Friendly" : "Restricted";
+	if(!hasSpaceport)
+		reputationLabel = "No Spaceport";
+	else
+	{
+		switch(planet->GetFriendliness())
+		{
+			case Planet::Friendliness::FRIENDLY:
+				reputationLabel = "Friendly";
```

Comment:
Shouldnt we have a default case; Friendly?
Also this seems unrelated but its fine